frontend/bower.json

```diff
@@ -12,7 +12,7 @@
     "drag-drop-polyfill": "2.0.0-beta.1",
     "pickadate": "^3.5.6",
     "At.js": "^1.5.4",
-    "airbrake-js-client": "airbrake-js#~0.5.8"
+    "airbrake-js-client": "airbrake-js#~1.4.2"
```

Comment:
I don't know what this might break but I think it's worth finding out. Not sure we use airbrake rn anyway - seems to be failing in prod https://airbrake.io/